### PR TITLE
2023-02-02 - wormhole-attack.md

### PR DESCRIPTION
**Pull request by akezhanshg**

### CODE REVIEW

File: content/attacks/posts/wormhole-attack [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+The attackers have not been identified. However, it is believed that they may be a group of hackers who have been targeting cryptocurrency projects in recent months.
+
+#Losses:
+
+The total value of cryptocurrency stolen in the attack is $32 million. This includes 120,000 ETH, which is worth $25 million, and 32,600 USDC tokens, which are worth $7 million.
+
+#Timeline:
+
+The attack took place on February 2, 2023. The Wormhole protocol team discovered the attack on February 3, 2023. The attackers were able to exploit a vulnerability in the protocol's code to steal the cryptocurrency.
```
Comment (jhirschkorn) [inline content/attacks/posts/wormhole-attack:25]: list the dates.  Get rid of the last sentence it doesnt add anything

**Comment by akezhanshg** [comment]
Ok got it

ср, 31 мая 2023 г., 23:15 Jules H ***@***.***>:

**Comment by jhirschkorn** [comment]
Conflict. Another article on this topic already exists.

**Comment by akezhanshg** [comment]
Ок

пт, 9 июн. 2023 г., 05:14 Jules H ***@***.***>:

**Comment by marina-chibizova** [comment]
@jhirschkorn closing then?

**Pull request closed without merging**